src/AudioAtom.tsx

```diff
@@ -116,6 +116,28 @@ const progressBarInputStyle = (pillar: Pillar) => css`
     :focus {
         ${focusHalo}
     }
+    // Use the pillar to style the colour of the range thumb
+    &::-webkit-slider-thumb {
+        background: ${pillarPalette[pillar][400]};
+        -webkit-appearance: none;
+        width: 14px;
+        height: 14px;
+        border-radius: 50px;
+    }
+    &::-moz-range-thumb {
+        background: ${pillarPalette[pillar][400]};
+        width: 14px;
+        height: 14px;
+        border: none;
+        border-radius: 50px;
+    }
+    &::-ms-thumb {
+        background: ${pillarPalette[pillar][400]};
+        width: 14px;
+        height: 14px;
+        border: none;
+        border-radius: 50px;
+    }
```

Comment:
```suggestion
    &::-webkit-slider-thumb, &::-moz-range-thumb, &::-ms-thumb {
        background: ${pillarPalette[pillar][400]};
        -webkit-appearance: none;
        width: 14px;
        height: 14px;
        border-radius: 50px;
    }
```

Reply:
In [this css-tricks article](https://css-tricks.com/styling-cross-browser-compatible-range-inputs-css/) it says you can't group these selectors like that:

> Note that while we’re repeating code here, that’s necessary as you can’t comma-separate these type of selectors. Browsers will drop the entire selector if it doesn’t understand a part of it.

I haven't tried this though

Reply:
cool! yes then you can ignore